packages/wallet-sdk/src/provider/CoinbaseWalletProvider.test.ts

```diff
@@ -1,6 +1,7 @@
 import { fireEvent } from "@testing-library/preact";
 
 import { MockRelayClass } from "../__mocks__/relay";
+import { standardErrorCodes, standardErrors } from "../errors";
```

Comment:
updated all the tests in this file that previously contained errors,
ensuring that each test correctly throws both error code and message